templates/JSON/treat_ad.json

```diff
@@ -0,0 +1,280 @@
+{
```

Comment:
I like the idea that @mattfazza mentioned on Slack where the expanded JSON output would be an array of top-level resources. For now, this would be multiple projects, but this could eventually include teams/users.